src/encoded/audit/experiment.py

```diff
@@ -262,11 +262,16 @@ def audit_experiment_missing_unfiltered_bams(value, system, files_structure):
 def audit_experiment_with_uploading_files(value, system, files_structure):
     if files_structure.get('original_files'):
         for file_object in files_structure.get('original_files').values():
-            if file_object['status'] in ['uploading', 'upload failed', 'content error']:
+            if file_object['status'] in ['upload failed', 'content error']:
                 detail = 'Experiment {} '.format(value['@id']) + \
                     'contains a file {} '.format(file_object['@id']) + \
                     'with the status {}.'.format(file_object['status'])
                 yield AuditFailure('file validation error', detail, level='INTERNAL_ACTION')
+            if file_object['status'] in ['uploading']:
+                detail = 'Experiment {} '.format(value['@id']) + \
```

Comment:
I realize this pattern for string formatting exists throughout the code and is copy/pasted.  It's hard to read and calls format 3 times instead of one.   

There are alternative ways that are shorter, faster, and easier to read.
```
detail = "Experiment {} contains a file {} with the status {}".format(
    value['@id'], file_object['@id'], file_object['status'])
```
or a little longer but easier to read and edit
```
detail = (
    "Experiment {value_id} contains a file {file_id} "
    "with the status {file_status}.".format(
        file_id=file_object['@id'],
        file_status=file_object['status'],
        value_id=value['@id'],
    )
)
```

Reply:
Fixed for this audit particularly, though like you say the problem spans the codebase.